### PR TITLE
bureau exceptions list

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺

### CODE REVIEW

File: seeds/data/bureau_exceptions_list.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        "PARM_VALUES": " ",
+        "BUREAU_NAME_LIST": " "
+      },
+      {
+        "HRU_ID": 21982,
+        "EMP_SEQ_NBR": null,
+        "EMP_FULL_NAME": "Pacheco, Reynaldo",
+        "PV_ID": -1,
+        "PARM_VALUES": " ",
+        "BUREAU_NAME_LIST": " "
```
Comment (elizabeth-jimenez) [inline seeds/data/bureau_exceptions_list.json:82]: notice this relationship here between having no Bureaus and PV_ID

**Comment by elizabeth-jimenez** [comment]
<head> 
<head> 
    <title> 
         “spoiler” text in github wiki pages 
    </title> 
</head> 
<body> 
    <details> 
        <summary>GeeksforGeeks</summary> 
         A Computer Science Portal for Geeks 
    </details>         
</body> 
</html>

File: seeds/data/bureau_exceptions_list_edit_view.json [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+      {
+        "ORG_CODE": "110000",
+        "ORGS_SHORT_DESC": "WHA"
+      }
+    ],
+    "O_RETURN_CODE": 0,
+    "QRY_ACTION_DATA": [],
+    "QRY_ERROR_DATA": []
+  },
+  "fail": {
```
Comment (elizabeth-jimenez) [inline seeds/data/bureau_exceptions_list_edit_view.json:203]: I could not get this Procedure to fail in a similar way to other ones i've found - this result is when i pass in 

{
  "PV_API_VERSION_I": "",
  "PV_AD_ID_I": ""
}

Comment (elizabeth-jimenez) [reply seeds/data/bureau_exceptions_list_edit_view.json:203]: If i pass in a fake i_emp_hru_id, then i get nulls for all the root values

  "O_EMP_HRU_ID": null,
  "O_EMP_FULL_NAME": "null",
  "O_PV_ID": null,
  "O_PV_VALUE_TXT": "null",
  "O_LAST_UPDATE_DATE": "null",
  "O_LAST_UPDATE_ID": null,

Comment (elizabeth-jimenez) [reply seeds/data/bureau_exceptions_list_edit_view.json:203]: verified all calls return the same list size for bureau ref

Comment (cagbakwu) [reply seeds/data/bureau_exceptions_list_edit_view.json:203]: Ah nice!

File: src/routes.js [hunk 3]
```diff
@@ -729,6 +732,9 @@ var appRouter = function (app) {
     "prc_lst_org_access": searchPostAccessList,
     "prc_lst_bureau_org_tree": searchPostAccessFilters,
     "prc_mod_org_access": searchPostAccessList,
+    "qry_lstbureauex": bureauExceptionsList,
+    "qry_getbureauex": bureauExceptionsListEditView,
+    "qry_addbureauex": bureauExceptionsListBureauRO,
     "qry_lstJobCats": jobCategories,
```
Comment (elizabeth-jimenez) [inline src/routes.js:738]: The last three procedures `act_modbureauex`, `act_addbureauex`, and `act_delbureauex`, the action ones, don't give a response worth mocking, so this PR is good to go

**Review by M00staff — DISMISSED** [review]

**Comment by elizabeth-jimenez** [comment]
image
i guess it auto does that on merge conflict resolutions 🤔

**Pull request merged**